src/Commands/Add.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Larowlan\Tl\Commands;
+
+use Larowlan\Tl\Connector\Connector;
+use Larowlan\Tl\Connector\ConnectorManager;
+use Larowlan\Tl\Formatter;
+use Larowlan\Tl\Repository\Repository;
+use Stecman\Component\Symfony\Console\BashCompletion\Completion\CompletionAwareInterface;
+use Stecman\Component\Symfony\Console\BashCompletion\CompletionContext;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ *
+ */
+class Add extends Command implements LogAwareCommand {
+
+  /**
+   * @var \Larowlan\Tl\Connector\Connector
+   */
+  protected $connector;
+
+  /**
+   * @var \Larowlan\Tl\Repository\Repository
+   */
+  protected $repository;
+
+  /**
+   *
+   */
+  public function __construct(ConnectorManager $connector, Repository $repository) {
+    $this->connector = $connector;
+    $this->repository = $repository;
+    parent::__construct();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this
+      ->setName('add')
+      ->setDescription('Add a time entry')
+      ->setHelp('Add a new entry, for a given that. <comment>Usage:</comment> <info>tl add [ticket number] [duration in hours]</info>')
+      ->addArgument('issue_number', InputArgument::REQUIRED, 'Issue number to start work on')
+      ->addArgument('duration', InputArgument::REQUIRED, 'Duration in hours to change slot to')
+      ->addArgument('comment', InputArgument::OPTIONAL, 'Comment to start with')
+      ->addOption('start', 's', InputOption::VALUE_OPTIONAL, 'A date offset', 'now')
+      ->addUsage('tl add 12355 0.5')
+      ->addUsage('tl add 12355 1 "Doin stuff"')
+      ->addUsage('tl add 12355 .25 "Daily Scrum" -s 11am')
+      ->addUsage('tl add 12345 1 -s 2018-12-31 11 am');
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $ticket_id = $input->getArgument('issue_number');
+    $duration = $input->getArgument('duration');
+    $start = $input->getOption('start');
+    if ($alias = $this->repository->loadAlias($ticket_id)) {
+      $ticket_id = $alias;
+    }
+    $connector_id = $this->connector->spotConnector($ticket_id, $input, $output);
+    if (!$connector_id) {
+      throw new \InvalidArgumentException('No such ticket was found in any backends.');
+    }
+    if ($alias = $this->connector->loadAlias($ticket_id, $connector_id)) {
+      $ticket_id = $alias;
+    }
+    if ($title = $this->connector->ticketDetails($ticket_id, $connector_id)) {
+      try {
+        $start_date_time = new \DateTime($start);
```

Comment:
this should be `try`d in a seperate block, and a unique message thrown if it could not be parsed.